src/messages/messages.service.ts

```diff
@@ -28,11 +35,19 @@ export class MessagesService {
       classroom,
     };
 
-    return this.messagesRepository.createOne(messageData);
+    const createdMessage = await this.messagesRepository.createOne(messageData);
+
+    this.socket.emit("sendMessage", {
```

Comment:
[QUESTION] Shouldn't we use `sendMessage` from  the `RealtimeMessagingGateway` to broadcast the message? Because it looks like nothing that was declared inside `RealtimeMessagingGateway` is being used.

Reply:
Yeah we are using "sendMessage" from the RealtimeMessagingGateway, you can see in line 40 I am using that, "this.socket.emit("sendMessage", ..."